src/pages/entropyInput.tsx

```diff
@@ -22,10 +22,10 @@ const EntropyInputPage = () => {
   const [mouseEntropy, setMouseEntropy] = useState('')
   const [percentage, setPercentage] = useState(0)
 
-  const updateEntropy = useEntropy((state: any) => state.updateEntropy)
+  const updateEntropy = useContributionStore((state: Store) => state.updateEntropy)
   const handleSubmit = () => {
     if (percentage !== 100) return
-    updateEntropy(entropy)
+    updateEntropy(0, entropy)    
```

Comment:
seems like having trailing whitespaces?

Reply:
I think it was my lint VS plugin x_x sorry